JPF/mysql.skeleton.ipynb

```diff
@@ -119,24 +119,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 117,
    "metadata": {
     "ExecuteTime": {
-     "end_time": "2016-11-18T12:28:05.962209",
-     "start_time": "2016-11-18T12:28:05.847715"
+     "end_time": "2016-11-21T10:20:53.398849",
+     "start_time": "2016-11-21T10:20:53.175839"
     },
     "collapsed": false
    },
    "outputs": [],
    "source": [
+    "MAX_COLUMN_LENGTH = 255\n",
+    "\n",
+    "\n",
     "def guess_sqlcol(dfparam):    \n",
     "\n",
     "## GUESS AT SQL COLUMN TYPES FROM DataFrame dtypes. \n",
     "    \n",
     "    dtypedict = {}\n",
     "    for i,j in zip(dfparam.columns, dfparam.dtypes):\n",
     "        if \"object\" in str(j):\n",
-    "            dtypedict.update({i: sqlalchemy.types.NVARCHAR(length=1024)}) ##big field length for HPD violations description\n",
+    "            dtypedict.update({i: sqlalchemy.types.NVARCHAR(length=MAX_COLUMN_LENGTH)}) ##big field length for HPD violations description\n",
```

Comment:
for the most part, not sure how much we will need the description .. would be better to have properly defined schema .. when I left it large, some tables with many varchar fields complained about too much data in the row. so far 255 has not caused issues .. but we need to be aware of potentially truncated data